includes/settings.php

```diff
@@ -306,7 +306,7 @@ function admin_menu() {
+	add_menu_page( 'Distributor', 'Distributor', 'manage_options', 'distributor-settings', __NAMESPACE__ . '\network_settings_screen', 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbDpzcGFjZT0icHJlc2VydmUiIHN0eWxlPSJmaWxsLXJ1bGU6ZXZlbm9kZDtjbGlwLXJ1bGU6ZXZlbm9kZDtzdHJva2UtbGluZWpvaW46cm91bmQ7c3Ryb2tlLW1pdGVybGltaXQ6MiIgdmlld0JveD0iMTMuNCA4LjggNTczLjIgNTczLjIiPgogIDxwYXRoIGZpbGw9IiNhN2FhYWQiIGQ9Ik0xOTUuMTEzIDQxMS4wMzNjNDUuODM1IDQ2LjY5MiAxMTkuMTI0IDU4LjQ4OCAxNzguMzg3IDI0LjI3MyA3MC4yNjItNDAuNTY2IDk0LjM3MS0xMzAuNTQ0IDUzLjgwNi0yMDAuODA2LTQwLjU2Ni03MC4yNjItMTMwLjU0NC05NC4zNzEtMjAwLjgwNi01My44MDYtMTkuODczIDExLjQ3NC0zNi4wNTUgMjYuODk5LTQ4LjEyNCA0NC43MTVsNjQuNzIyIDMzLjE4NmMyMi4yMDEtMjUuNTkzIDU5Ljc5Ni0zMy43ODIgOTEuMjc5LTE3LjYzOSAzNy4wMDIgMTguOTczIDUxLjY0IDY0LjQxOCAzMi42NjcgMTAxLjQyMS0xOC45NzMgMzcuMDAyLTY0LjQxOCA1MS42NC0xMDEuNDIxIDMyLjY2Ny0zMS40ODMtMTYuMTQzLTQ2Ljc3Ni01MS40NS0zOC45NTEtODQuNDE1bC04MS43MDItNDEuODkyYy04LjgzOC00LjUzMi0xMi4zMzUtMTUuMzY3LTcuODE0LTI0LjIxMSAxNS41MTQtMzAuMzQ2IDM5LjY1OC01Ni43MTUgNzEuMzQ0LTc1LjAwOSA4Ny40NjktNTAuNSAxOTkuNDgyLTIwLjQ4NiAyNDkuOTgzIDY2Ljk4MyA1MC41IDg3LjQ2OSAyMC40ODYgMTk5LjQ4Mi02Ni45ODMgMjQ5Ljk4My03NS4yMzUgNDMuNDM3LTE2OC42MyAyNy4zMDctMjI1LjQxOS0zMy43MTctMTcuODA5IDMuNzc4LTM2Ljc5Ny00LjA1NS00Ni4zODctMjAuNjY2LTExLjkyMi0yMC42NDgtNC44MzctNDcuMDkxIDE1LjgxMi01OS4wMTIgMjAuNjQ4LTExLjkyMiA0Ny4wOTEtNC44MzYgNTkuMDEyIDE1LjgxMiA3Ljc3IDEzLjQ1OCA3LjQ2NiAyOS4zNzcuNTk1IDQyLjEzM1oiLz4KICA8cGF0aCBmaWxsPSIjYTdhYWFkIiBkPSJNMjYyLjIzNyA3Mi45ODVDMTQ4LjggOTEuMTAxIDYyIDE4OS40OTQgNjIgMzA4YzAgMTMxLjM1NiAxMDYuNjQ0IDIzOCAyMzggMjM4czIzOC0xMDYuNjQ0IDIzOC0yMzhjMC0zNC4wNTktNy4xNjgtNjYuNDU4LTIwLjA4LTk1Ljc2Ni0xNS4xMjEuOTktMzAuMzIzLTYuMDE0LTM5LjEzNy0xOS42MjYtMTIuOTU5LTIwLjAxNC03LjIzMS00Ni43ODMgMTIuNzgzLTU5Ljc0MiAyMC4wMTQtMTIuOTU4IDQ2Ljc4My03LjIzMSA1OS43NDIgMTIuNzgzIDEwLjA5NSAxNS41OTIgOC44NDkgMzUuMjg0LTEuNjU3IDQ5LjM1MkM1NjUuMjg4IDIyOS40NjEgNTc0IDI2Ny43MjEgNTc0IDMwOGMwIDE1MS4yMjUtMTIyLjc3NSAyNzQtMjc0IDI3NFMyNiA0NTkuMjI1IDI2IDMwOEMyNiAxNzAuNTM5IDEyNy40NDMgNTYuNTg0IDI1OS40ODcgMzYuOTggMjY1LjU5NCAyMC41MzMgMjgxLjQzOCA4LjggMzAwIDguOGMyMy44NDMgMCA0My4yIDE5LjM1NyA0My4yIDQzLjIgMCAyMy44NDMtMTkuMzU3IDQzLjItNDMuMiA0My4yLTE2LjIyOSAwLTMwLjM4LTguOTY4LTM3Ljc2My0yMi4yMTVaIi8+Cjwvc3ZnPgo=' );
```

Comment:
@zach-adams can we create a separate file and function to get the data image

Reply:
Sure, I'll take a look

Reply:
@ravinderk Would moving these data URL strings to a function in utils.php called something like "get_admin_icon" work alright? That way it could be simplified both here and in the 'external-connection-cpt.php' file to:
```suggestion
	add_menu_page( 'Distributor', 'Distributor', 'manage_options', 'distributor-settings', __NAMESPACE__ . '\network_settings_screen', Utils\get_admin_icon() );
```

Reply:
@zach-adams Make sense.